tendermint/src/account.rs

```diff
@@ -12,7 +12,8 @@ use subtle::{self, ConstantTimeEq};
 use subtle_encoding::hex;
 
 /// Size of an  account ID in bytes
-const LENGTH: usize = 20;
+// TODO: Is it okay to publicize this??
```

Comment:
Sure, why not. Especially as it's a const it can't hurt. It only means that this is more exposed users of the API. It was implicitly accessible anyways...